config/software/z_calibration.cfg

```diff
@@ -8,5 +8,13 @@ switch_xy_position: 198.5,285.5
 switch_offset: 0.390
 max_deviation: 0.750
 samples: 3
+#samples_tolerance: 0.01
+#clearance: 8
 speed: 350 # X,Y movements
 probing_first_fast: true
+#probing_speed: 20
+#probing_second_speed: 5
+#probing_retract_dist: 2
+start_gcode: ATTACH_PROBE
```

Comment:
Is this working OK ?
As you did not make any change to the macros to remove the call to `ATTACH/DOCK`, so maybe there is some place where it's not needed anymore if we keep it here ?

Reply:
This is for docking probe after a probing error.
(updated in v0.9.2 of plugin Z-calibration: _The probe is now docked again before raising an error, if end_gcode is used (thanks to top-gun)_

I'm agree as we lock probe in start_print this may occured only when we are working in manual

Reply:
Mhmm OK. It's a good thing indeed to dock the probe in case of an error !
Ok, if you tested it and it's working correctly (I mean no double attach move or double detach move), we can add it now. But I'll try to investigate it more in deep to understand every possible case and do it properly.